### ISSUE TITLE
Docker compose fails to build the project end to end on "Rounding Decimals" error

### ISSUE THREAD

**Issue by sverchdotgov** [usds/justice40-tool#1108]
**Describe the bug**
The docker compose workflow described in the [quickstart](https://github.com/usds/justice40-tool/blob/main/QUICKSTART.md) doesn't work.

**To Reproduce**

```
$ docker-compose up --build
j40_data_pipeline_1 | 2022-01-06 08:47:25,574 [data_pipeline.etl.score.etl_score_post] INFO     Rounding Decimals
j40_data_pipeline_1 | Traceback (most recent call last):
j40_data_pipeline_1 |   File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
j40_data_pipeline_1 |     return _run_code(code, main_globals, None,
j40_data_pipeline_1 |   File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
j40_data_pipeline_1 |     exec(code, run_globals)
j40_data_pipeline_1 |   File "/data-pipeline/data_pipeline/application.py", line 283, in <module>
j40_data_pipeline_1 |     cli()
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1128, in __call__
j40_data_pipeline_1 |     return self.main(*args, **kwargs)
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1053, in main
j40_data_pipeline_1 |     rv = self.invoke(ctx)
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1659, in invoke
j40_data_pipeline_1 |     return _process_result(sub_ctx.command.invoke(sub_ctx))
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1395, in invoke
j40_data_pipeline_1 |     return ctx.invoke(self.callback, **ctx.params)
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 754, in invoke
j40_data_pipeline_1 |     return __callback(*args, **kwargs)
j40_data_pipeline_1 |   File "/data-pipeline/data_pipeline/application.py", line 266, in data_full_run
j40_data_pipeline_1 |     score_post(data_source)
j40_data_pipeline_1 |   File "/data-pipeline/data_pipeline/etl/runner.py", line 102, in score_post
j40_data_pipeline_1 |     score_post.transform()
j40_data_pipeline_1 |   File "/data-pipeline/data_pipeline/etl/score/etl_score_post.py", line 258, in transform
j40_data_pipeline_1 |     self.output_score_tiles_df = self._create_tile_data(
j40_data_pipeline_1 |   File "/data-pipeline/data_pipeline/etl/score/etl_score_post.py", line 211, in _create_tile_data
j40_data_pipeline_1 |     score_tiles[constants.TILES_SCORE_FLOAT_COLUMNS] = score_tiles[
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 3600, in __setitem__
j40_data_pipeline_1 |     self._setitem_array(key, value)
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 3641, in _setitem_array
j40_data_pipeline_1 |     self[k1] = value[k2]
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 3612, in __setitem__
j40_data_pipeline_1 |     self._set_item(key, value)
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 3797, in _set_item
j40_data_pipeline_1 |     self._set_item_mgr(key, value)
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 3762, in _set_item_mgr
j40_data_pipeline_1 |     self._check_setitem_copy(stacklevel=5)
j40_data_pipeline_1 |   File "/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py", line 3933, in _check_setitem_copy
j40_data_pipeline_1 |     raise com.SettingWithCopyError(t)
j40_data_pipeline_1 | pandas.core.common.SettingWithCopyError: 
j40_data_pipeline_1 | A value is trying to be set on a copy of a slice from a DataFrame.
j40_data_pipeline_1 | Try using .loc[row_indexer,col_indexer] = value instead
j40_data_pipeline_1 | 
j40_data_pipeline_1 | See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
j40_data_pipeline_1 exited with code 1
```

**Expected behavior**
The [quickstart](https://github.com/usds/justice40-tool/blob/main/QUICKSTART.md) can create and run the application.

**Comment by sverchdotgov** [comment]
@saran-ahluwalia Assigned you because I think [this commit](https://github.com/usds/justice40-tool/commit/56644698ff40b0a155ef1f872cd798bb3462f25f) introduced the bug.

@esfoobar-usds I'm assuming that our github actions resources aren't enough to run this end to end test, but maybe there's something that could be done to catch cases like this. Do you think this is worth looking into? I can file a follow up issue if so.

**Comment by saran-ahluwalia** [comment]
Based on independently running `score-post` and `score-run` locally does not trigger this exception. @esfoobar-usds was able to corroborate this result. @esfoobar-usds did eliminate the `SettingWithCopyWarning` by creating a new reference pointer to the series object further up in the call chain.

**Comment by esfoobar-usds** [comment]
Just ran this against my branch and it runs successfully.

**Comment by sverchdotgov** [comment]
I think this is still broken in `main`, maybe you accidentally hit "comment and close".

**Comment by sverchdotgov** [comment]
This was unrelated to @saran-ahluwalia's change. The current theory is that this was a pandas warning that turned into an error when we updated dependencies (because that's the way pandas gradually phases out usage).

@esfoobar-usds has a fix for the underlying issue on his branch (we've confirmed that works), so once his branch is merged in, we can close this. The branch is: https://github.com/usds/justice40-tool/tree/esfoobar-usds/1062-implement-changes-export-files
